MAUI/Themes/Themes.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Applying themes for Syncfusion controls
+description: This section explains about theming in Syncfusion controls and how to override the key values for further customization
+platform: maui
+control: General
+documentation: ug
+---
+
+# Applying Themes in Maui Controls
+
+Syncfusion themes allow you apply colors across all the Syncfusion controls with a uniform approach and provide consistent look and feel to your applications. This section covers the following topics: 
+
+* Applying light and dark themes
+* Overriding the default theme
+* Creating your own theme
+
+## Applying light and dark themes
+
+By default, Syncfusion provides support for light and dark themes. As names suggest, these themes will have colors with light and dark color contrasts, respectively.
+
+To apply themes to your application, merge the following items:
+
+* Theme resource dictionary
+
+#### Theme resource dictionary
+
+This resource dictionary contains *keys* and their respective color codes for all the Syncfusion controls. Currently, the following two theme resource dictionaries are available:
```

Comment:
Explain this section properly. say about SfVisual property.

Reply:
Changes Addressed.